content-services/content-sources-frontend
# Fixes 4488: Add upload custom content modal

## Summary
This adds the following modal: 

![Screenshot 2024-09-06 at 12 04 13 PM](https://github.com/user-attachments/assets/ececc2bd-23a6-4694-b7c2-e0e28cde9db9)
![Screenshot 2024-09-06 at 12 04 18 PM](https://github.com/user-attachments/assets/199a7149-7e79-452f-b24c-2055d1a70674)
![Screenshot 2024-09-06 at 12 04 22 PM](https://github.com/user-attachments/assets/ff70d82f-1cac-487d-b6ed-dac377702808)
![Screenshot 2024-09-06 at 12 03 54 PM](https://github.com/user-attachments/assets/8de57315-1778-48ab-b8a2-de47f37f0d19)


## Testing steps
- Create a upload repository (split buttons added in this PR @swadeley)
- Navigate to the upload modal via the addModal button or the kebab in the contentList. 
- Upload a few rpms. 
- Confirm changes.
